### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [Portfolio: setup and mobile version skeleton](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

### These are the changes proposed to the Portfolio project:

- Create all files to set up Github workflows and linters.
- Create a mobile version skeleton from Figma ([template 3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <a href="#">
+          <img src="/images/Icon-Menu.png" alt="hamburger-icon">
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2 class="name">
+          Hey there. I'm <br> Marco
+        </h2>
```
Comment (cwaku) [inline index.html]: - [x] Kindly change this to `h1`. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          <li><a href="#"><img src="/images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/Linkedin-icon.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/medium.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/github.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/angellist.png" alt=""></a></li>
+        </ul>
+        <a class="down-icon" href="#">
+          <img src="/images/down.png" alt="down-icon">
+        </a>
+      </div>
```
Comment (cwaku) [inline index.html]: - [x] It is a core requirement that you align the content in this section with the flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. Kindly fix that

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC 

Good job so far! Your project is beautiful.
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
- [x] Very good overall template design 🥇 
- [x] Linters passed 👍🏾 
- [x] Meaningful PR description and title 💯 
- [x] Correvt use of Github flow ✔️ 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MarcoOquendoC) [reply index.html]: # Hi @cwaku,

## I made these changes:

- Turn h2 to h1 as you asked to make it the most important header on the web page.
- Add a class attribute to some tags.
- Change to flexbox the section.
- Move inside the flexbox all the content.

**Review by isaacpitwa — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @MarcoOquendoC  👋🏻 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**